waspc/src/Wasp/Util/ShellCommand.hs

```diff
@@ -0,0 +1,22 @@
+module Wasp.Util.ShellCommand where
+
+import Data.String (IsString (fromString))
+
+data ShellCommandArg = Raw String | Quoted String
+  deriving (Eq)
+
+instance Show ShellCommandArg where
+  show (Raw arg) = arg
+  show (Quoted arg) = "\"" ++ arg ++ "\""
+
+instance IsString ShellCommandArg where
+  fromString = Raw
+
+showShellArgs :: [ShellCommandArg] -> String
+showShellArgs = unwords . map show
```

Comment:
I would drop this one, it doesn't do much, I can just call `unwords . map show` myself if I want and at least I know what it is doing then. Beacuse this not only shows the args but also concatenates them and that is not clear from the name.
So better not to have this function.